docs/source/dashboard.rst

```diff
@@ -53,6 +53,25 @@ of the most commonly used plots shown on the entry point for the dashboard:
 
 Bytes Stored and Bytes per Worker
 ---------------------------------
+
+The following code is given to show the setting of the following example.
+
+ .. code-block:: python
+
+    import dask
+    import distributed
+
+    dask.config.set(
+        {
+            "distributed.worker.memory.max_spill": "250 MiB",
+            "distributed.worker.memory.terminate": False,
+        }
+    )
+
+    c = distributed.Client(
+        threads_per_worker=4, n_workers=4, memory_limit="4 GiB"
+    )
+
```

Comment:
I don't see the purpose of this code snippet; I think it's needlessly confusing for the reader and doesn't add any value.

Reply:
Okay, @crusaderky , removing the code snippet from there.